src/ILVerification/src/AccessVerificationHelpers.cs

```diff
@@ -250,20 +250,38 @@ private static bool CanAccessFamily(TypeDesc currentType, TypeDesc targetTypeDef
             return false;
         }
 
-        private static bool GrantsFriendAccessTo(this ModuleDesc module, ModuleDesc friendModule)
+        private static EcmaAssembly ToEcmaAssembly(this ModuleDesc module)
         {
-            var assembly = (EcmaAssembly)module;
-            var friendName = ((IAssemblyDesc)friendModule).GetName();
-
-            foreach (var attribute in assembly.GetDecodedCustomAttributes("System.Runtime.CompilerServices", "InternalsVisibleToAttribute"))
+            EcmaAssembly ecmaAssembly = module as EcmaAssembly;
```

Comment:
This can be simplified to just:

```
private static EcmaAssembly ToEcmaAssembly(this ModuleDesc module)
{
    return module.Assembly as EcmaAssembly'
}
```